src/main/kotlin/kotlinx/kover/KoverPlugin.kt

```diff
@@ -166,6 +178,48 @@ private fun Task.jacocoReport(
     }
 }
 
+
+fun Task.jacocoVerification(builder: GroovyObject, extension: KoverTaskExtension) {
```

Comment:
`KoverPlugin.kt` is becoming a jack of all trades, could you please split it up a bit?

E.g. all verification mapping can go to a separate `Verification.kt` file and so on